digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/ServiceRoadAPI.scala

```diff
@@ -23,14 +52,40 @@ class ServiceRoadAPI(val maintenanceService: MaintenanceService, val roadLinkSer
     basicAuth
   }
 
-  get("/huoltotiet"){
+  val getServiceRoadByBoundingBox =
+    (apiOperation[List[serviceRoadApiResponseOnGetExample]]("getServiceRoadByBoundingBox")
+      tags "Service Road API (Huoltotie API)"
+      summary "Returns all Huoltotie assets inside bounding box. Can be used to get all assets on the UI map area."
+      parameter queryParam[String]("boundingBox").description("The bounding box is used to search assets inside him, is defined with coordinates of top left and bottom right corner.")
+      description
+      "Bounding box is defined with coordinates of top left and bottom right corner \n" +
+        "URL: /digiroad/api/livi/huoltotiet/?boundingBox={x1},{y1},{x2},{y2} \n" +
+        "Example: \n https://extranet.liikennevirasto.fi/digiroad/api/livi/huoltotiet?boundingBox=399559.02383961395,6856819.997802734,401097.02383961395,6858507.997802734"
+      )
+
+
+  get("/huoltotiet", operation(getServiceRoadByBoundingBox)){
     contentType = formats("json")
     val bbox = params.get("boundingBox").map(constructBoundingRectangle).getOrElse(halt(BadRequest("Bounding box was missing")))
     validateBoundingBox(bbox)
     createGeoJson(maintenanceService.getAllByBoundingBox(bbox))
   }
 
-  get("/huoltotiet/:areaId"){
+  val getServiceRoadByAreaId =
+    (apiOperation[List[serviceRoadApiResponseOnGetExample]]("getServiceRoadByAreaId")
+      tags "Service Road API (Huoltotie API)"
+      summary "Returns all Huoltotie assets inside service area."
+      parameter pathParam[String]("areaId").description("The area id to search assets inside him.")
```

Comment:
`inside him` ->` inside it` or something like `Area id refers to the area where the search is going to be done`